src/OrleansRuntime/OrleansRuntime.csproj

```diff
@@ -186,7 +186,6 @@
     <Compile Include="Streams\QueueBalancer\StaticClusterDeploymentConfiguration.cs" />
     <Compile Include="Streams\QueueBalancer\ConsistentRingQueueBalancer.cs" />
     <Compile Include="Streams\QueueBalancer\DeploymentBasedQueueBalancer.cs" />
-    <Compile Include="Streams\QueueBalancer\IDeploymentConfiguration.cs" />
```

Comment:
Why move this?
Why associate queue balancer with persistent streams.  The persistent streams relies on the queue balancer.  Queue balancer does not rely on persistent streams.  There is nothing limiting queue balancing to streaming.